Editor/Window/ElementLocalizer.cs

```diff
@@ -27,6 +27,12 @@ public void SetElementText(string elementName, string textKey)
             var text = _textProvider.Get(textKey);
             SetText(elementName, text);
         }
+        
+        public void SetElementText(string elementName, string textKey, string additionalText)
```

Comment:
It's not evident to me from reading this what additionalText is supposed to do, can you maybe add a comment on top with context to when it should be used to make things clear - for example: the status box use case where you want to display a Exception message in addition to the generic error text

Reply:
Added Comment